project-movies on request Mathilda

Code-lab-web opened the pull request:
Please include your Netlify link here.

Code-lab-web committed:
Refactor application structure and add routing for Movie Company

Code-lab-web committed:
Update .gitignore, modify button text, enhance CSS styles, and add Movies component for fetching popular movies

Code-lab-web committed:
Update dependencies, refactor About and Movies components, and add PopularList page

Code-lab-web committed:
Fix import statement for React to use react-router-dom

Code-lab-web committed:
Code-lab-web

Code-lab-web committed:
Add Rollup external configuration for styled-components in Vite

Code-lab-web committed:
Update package.json to add hooks dependency and upgrade Vite version

Code-lab-web committed:
Add react-router dependency to package.json

Code-lab-web committed:
Add About, Movies, and Header components; update App component with routing

Code-lab-web committed:
Refactor App component structure; add MovieDetails and NotFound pages; implement Card component for movie display; enhance movie fetching logic; update styles and remove unused Header component

Code-lab-web committed:
Refactor Movies component; remove unused variable 'apikey'; update fetch URL to use template literals; improve error handling and loading state

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
Refactor API calls in tests; remove invalid lines; integrate image configuration object; enhance image URL handling

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
Code-lab-web

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
Code-lab-web

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
Refactor import paths for pages in App.jsx

Code-lab-web committed:
Fix syntax errors and improve component structure in Contact, Movies, Nav, and Welcome components

Code-lab-web committed:
Fix API endpoint in Movies component for fetching popular movies

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
Merge pull request #1 from Code-lab-web/draft/objective-dijkstra

git commit -m "Code-lab-web"

Code-lab-web committed:
Fix API key usage in Movies component and clean up CSS styles

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
Add dotenv dependency for environment variable management

Code-lab-web committed:
Add redirects configuration for improved URL management

Code-lab-web committed:
Add express dependency and clean up main.jsx file

Code-lab-web committed:
Add movie data JSON and implement server-side API call for authentication

Code-lab-web committed:
Add Material-UI and styled-components dependencies

Code-lab-web committed:
Add @fontsource/roboto dependency for improved typography

Code-lab-web committed:
Add Material-UI icons and Roboto font styles for enhanced UI

Code-lab-web committed:
Update react-dom dependency to version 19.1.0

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit-m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

JennieDalgren reviewed (changes requested):
Read the requirements for this project and follow the guide on how to work with github! It is not possible to review this at all.

You are not approved on this project

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m ""

Commit:
git commit -m "Code-lab-web"

HIPPIEKICK commented:
ping

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
Create SECURITY.md for security policy

Add a security policy document outlining supported versions and vulnerability reporting.

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
Add CRDA static analysis workflow

This workflow performs static analysis of source code using Red Hat CodeReady Dependency Analytics, triggered on pushes, pull requests, and schedules.

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
git commit -m "Code-lab-web"
